crates/flox-rust-sdk/src/models/project/mod.rs

```diff
@@ -8,7 +8,7 @@ use runix::command::{Eval, FlakeInit};
 use runix::flake_ref::git::{GitAttributes, GitRef};
 use runix::flake_ref::indirect::IndirectRef;
 use runix::flake_ref::FlakeRef;
-use runix::installable::Installable;
+use runix::installable::{FlakeAttribute, Installable};
```

Comment:
Do we still have `Installable` values here?

Reply:
Yeah an argument to `init_flox_package` is still an `Installable`